src/analyze-trace-file.ts

```diff
@@ -338,12 +344,20 @@ async function makePrintableTree(curr: EventSpan, currentFile: string | undefine
 
     return childTree;
 
+    // NB: May alter childTree
     async function eventToString(): Promise<string | undefined> {
         const event = curr.event!;
         switch (event.name) {
             // TODO (https://github.com/microsoft/typescript-analyze-trace/issues/2)
-            // case "findSourceFile":
-            //     return `Load file ${event.args!.fileName}`;
+            case "findSourceFile":
+                const sourcePath = event.args!.fileName;
+                const childCount = curr.children.length + curr.droppedChildCount;
+                if (childCount < sourceFileFanoutThreshold || !/node_modules/.test(sourcePath) || /@types\/node/.test(sourcePath)) {
```

Comment:
In user code, it's definitely uninteresting.  `@types/node` seemed special enough to get an exception too, but it turns out that there are many exceptions - more than true positives, probably.